src/main/scala/org/dbpedia/spotlight/wikistats/ComputeStats.scala

```diff
@@ -45,14 +44,14 @@ class ComputeStats(lang: String) (implicit val sc: SparkContext,implicit val sql
 
   def buildCounts(wikipediaParser: JsonPediaParser,stopWordLoc: String): (DataFrame, DataFrame)={
 
-
     val allSfs = wikipediaParser.getSfs().collect().toList
-
+    println ("All sfs count")
```

Comment:
Yes. This is a good idea. I am making a note of this and will work on it once I complete the current task of generating raw wiki text.